discovery_child_development/notebooks/00_preprocess_openalex.py

```diff
@@ -0,0 +1,188 @@
+# %% [markdown]
+# This notebook is for preprocessing the Openlex data we have extracted and turning it into a format that can be the input to some text clustering.
+#
+# The data was generated by running `discovery_child_development/pipeline/openalex_workflow.py` from the command line via
+# ```
+# python discovery_child_development/pipeline/openalex_workflow.py run --production=True
+# ```
+#
+# Two outputs are saved:
+# * the larger of the two is a table that includes in long format every concept that is associated with each OpenAlex ID and their scores
+# * the smaller is a table with one row per work, containing the OpenAlex ID, title, abstract (deinverted) and "text" which is simply title + ' ' + abstract.
+
+# %%
+import json
+import os
+import boto3
+from dotenv import load_dotenv, find_dotenv
+import pandas as pd
+from io import StringIO
+
+# %%
+# We need to change wd in order to import utils functions
+env_path = find_dotenv()
+env_dir = os.path.dirname(env_path)
+os.chdir(env_dir)
+os.getcwd()
+
```

Comment:
Hey, no need for this. You just need to run one the following command from the repo's root directory:

```shell
pip install -e .
```

And this will install your repo as another package. So, once you run the pip command, you can delete these lines and simply import stuff from `discovery_child_development` as it's another package.

I'd suggest adding this in the repo's README, thanks!